src/docs/introduction.rst

```diff
@@ -1,12 +1,102 @@
+This isn’t necessary when you have a handful of machines to manage but
+when scaled up to tens, hundreds, and thousands something needs to be
+in place. You certainly can force the new guy to go to each machine to
+create a user, update their password, remove a user. However, that’s
+pretty mean and a waste of time when there is a much easier way of
+doing things.
+
+The community project FreeIPA *(Free Identity Policy and Auditing)* is
+an Identity Management solution. It’s also known as Red Hat IdM, which
+is the downstream name. FreeIPA, Red Hat IdM uses 389DS
+*(Directory Server)* for its database.
+
+These directory servers use the LDAP *(Lightweight Directory Access Protocol)*
+network protocol to representing objects to the computers on the network.
+Objects like a user, then that user object contains his user information
+like given name, surname, phone number, etc.
+
+*Please do not confuse LDAP with 389DS, 389DS is a directory server that
+uses the LDAP protocol on port 389.*
+
+Microsoft *(Active Directory Domain Services)* ADDS, but abbreviated to
+Microsoft AD. AD is another identity management solution for Windows. It has
+an LDAP server and shares many components with IPA. IPA and AD can have
+a trust created between the domain domains. Why? Consider if there was a
+company merger or acquisition. There won’t be a need to migrate users but
+instead, set up a trust and allow Bob from RedHat to administer
+the IBM domain…
+
+It’s not uncommon for companies to have two departments, one that supports
+and administers Linux and the other team does Windows, so each will have
+control over its own users and groups. Using SSSD’s ipa_provider,  AD domain
+users can then log in to a host managed by IPA because of the established
+trust between domains.
+
+SSSD can authenticate directly to AD, using the ad_provider, and they’ll
+be no need for IPA servers. Users and groups will be entirely managed by
+AD. So why use indirect authentication? IPA is more flexible for managing
+Linux hosts than AD since it can manage SUDO rules, SELinux users and
+contexts, automount maps, and more. If all users are created in AD, but
+the company has two IT teams. The Linux team may have to wait for users
+to be created. Hopefully, you can start to see that each solution has its
+pros and cons and will not be discussed in this introduction.
+
+For more information about FreeIPA and other compatible directory servers,
+please check out the following links.
+
+- `FreeIPA <https://www.freeipa.org/page/Main_Page>`_
```

Comment:
I think this should be merged with 1st chapter where you enumerate all those names.

Reply:
I disagree, we haven't established what a directory is yet and we are sending people to click those links early, @justin-stephenson deciding vote?

Reply:
I don't have much preference, i'd be more inclined to include the links towards the bottom of the article(like a footer)

Reply:
True, after 2nd reading I agree that hitting reader with the links at the beginning is bad idea.